aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/migrations/AddSyncExpressionToLastSyncMetadata.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.datastore.storage.sqlite.migrations;
+
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+
+import com.amplifyframework.core.Amplify;
+import com.amplifyframework.core.category.CategoryType;
+import com.amplifyframework.logging.Logger;
+import com.amplifyframework.util.Wrap;
+
+/**
+ * Add SyncExpression (TEXT) column to LastSyncMetadata table.
+ */
+public final class AddSyncExpressionToLastSyncMetadata implements ModelMigration {
```

Comment:
Should not be public

Reply:
Make sense! I had addressed this in my latest commit